modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js

```diff
@@ -116,17 +116,9 @@ export function FormBuilder() {
 	);
 
 	useEffect(() => {
-		const sessionLength = Liferay.Session
-			? Liferay.Session.get('sessionLength')
-			: 60000;
+		Liferay.Session.set('autoExtend', true);
 
-		const interval = setInterval(() => {
-			if (Liferay.Session) {
-				Liferay.Session.extend();
-			}
-		}, sessionLength / 2);
-
-		return () => clearInterval(interval);
+		return () => Liferay.Session.set('autoExtend', false);
```

Comment:
I don't know if there's a way to the user set the `autoExtend` to `true`, but just in case we should get the original value and to restore that in the end:
```javascript
const autoExtend = Liferay.Session.get('autoExtend');

if(!autoExtend) {
    Liferay.Session.set('autoExtend', true);
    return () => Liferay.Session.set('autoExtend', autoExtend);
}
```